config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -0,0 +1,286 @@
+# rubocop:disable Layout/LineLength
```

Comment:
What is the purpose of the entire new file of all commented out lines?

Reply:
it's documentation of all the new defaults! i think it can be useful if something isn't working as expected to be able to see all the new defaults in one place in the code, but happy to remove it if that doesn't feel useful to folks.

Reply:
I'm torn. I like the idea of having them, but am not sold that the codebase is the place for it. If we had Wiki's enabled on this repo that'd be where I'd like to see it. I feel like confluence is too much of a step removed though. I guess leaving it in place is the best option.